test/pylib/scylla_cluster.py

```diff
@@ -200,6 +201,15 @@ class CqlUpState(Enum):
     CONNECTED = 2,
     QUERIED = 3
 
+
+def start_stop_lock(func):
+    async def wrap(self, *args, **kwargs):
```

Comment:
please add type hint to `self`: `self: ScyllaServer`

Reply:
fixed